src/_shared/scss/_core.scss

```diff
@@ -182,8 +190,17 @@ $mq-breakpoints: (
 
 .justify--start { justify-content: flex-start; }
 
-// ----- Gimbap ----- //
+// Gimbap fonts
 
+@mixin f-headline {
```

Comment:
Ah good catch. Some of the mixins he removed were being used for the multiple, so I merged those changes and went through replacing all the instances. Looks like I missed that `f-headline`!